src/wio.zig

```diff
@@ -113,6 +113,16 @@ pub const Window = struct {
         self.backend.setCursorMode(mode);
     }
 
+    // TODO: macos
+    pub fn setSize(self: *Window, size: Size) void {
+        self.backend.setSize(size);
+    }
+
+    // TODO: macos
+    pub fn getSize(self: *Window) Size {
+        return self.backend.getSize();
+    }
```

Comment:
wio generally doesn't have functions that provide the same information as events.